updating/updating-cluster-prepare.adoc

```diff
@@ -1,16 +1,16 @@
 :_content-type: ASSEMBLY
 [id="updating-cluster-prepare"]
-= Preparing to update to {product-title} 4.12
+= Preparing to update to {product-title} 4.13
```

Comment:
Can you back out the changes to this file? I have another PR that will take care of everything needed in this file (#55224)